recipes/pygnome/meta.yaml

```diff
@@ -0,0 +1,85 @@
+{% set name = "PyGnome" %}
+{% set version = "1.1.11" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/NOAA-ORR-ERD/PyGnome/archive/v{{ version }}.tar.gz
+  sha256: f4263ea7e373414ea66ca0f630b705b07e0131258b2fafad03d369b0f5ef0ee6
+
+build:
+  skip: true  # [linux]
+  number: 0
+  script: cd py_gnome && {{ PYTHON }} -m pip install ./ -vv
+
+requirements:
+  build:
+    - cython
+    - numpy
+    - netCDF4
+    - scikit-build-core
+    - clang 15.*
+    - cmake
+  host:
+    - python >=3.7
+    - pip
+    - numpy
+    - scikit-build-core
+  run:
```

Comment:
this is where the full list of requirements should go -- not in the tests stanza.

Reply:
So they should go in the requirements/run?

Reply:
Exactly -- `requirements/run` is what is need to, well, run the package -- that's what will get installed by conda when you conda install this package.